app/routes.js

```diff
@@ -30,4 +30,23 @@ exports.init = app => {
     ],
     userController.userCreate
   );
+
+  app.post(
+    '/users/sessions',
+    [
+      check('password')
```

Comment:
Same as the previous PR this validation params should be moved to a different file inside the `middlewares` folder to clean up a little bit the route file. But let's do this in another branch and PR.

Reply:
Done in #13